sched/worker/workers/makers.go

```diff
@@ -5,25 +5,28 @@ import (
 
 	"github.com/scootdev/scoot/cloud/cluster"
 	"github.com/scootdev/scoot/os/temp"
+	"github.com/scootdev/scoot/runner"
 	"github.com/scootdev/scoot/runner/execer/execers"
 	"github.com/scootdev/scoot/runner/runners"
-	"github.com/scootdev/scoot/sched/worker"
 	"github.com/scootdev/scoot/snapshot/snapshots"
 )
 
 // Makes a worker suitable for using as an in-memory worker.
-// TODO: these are useful for unit tests but really shoudldn't be used for integration tests...
-func MakeInmemoryWorker(node cluster.Node, tmp *temp.TempDir) worker.Worker {
+func MakeInmemoryWorker(node cluster.Node, tmp *temp.TempDir) runner.Service {
```

Comment:
If snapshotId is set then workers should return an appropriate error? I'm not sure if there's a good way to implement my suggestion.